### PR TITLE
Seafile updates

### PR DESCRIPTION
**Pull request by evdenis**
Python 3.7 support && version bump to 7.0.7

### CODE REVIEW

File: net-libs/libsearpc/libsearpc-3.2.0-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+EAPI=7
+
+PYTHON_COMPAT=(python3_{7,8})
+
+WANT_AUTOMAKE=1.16
+
+inherit autotools python-single-r1
+
+DESCRIPTION="A simple C language RPC framework"
+HOMEPAGE="https://github.com/haiwen/libsearpc/ http://seafile.com/"
```
Comment (charIes17) [inline net-libs/libsearpc/libsearpc-3.2.0-r2.ebuild:13]: https also for https://www.seafile.com/

Comment (evdenis) [reply net-libs/libsearpc/libsearpc-3.2.0-r2.ebuild]: Fixed

File: net-misc/seafile-client/Manifest [hunk 2]
```diff
@@ -1,4 +1,3 @@
...
 DIST seafile-client-7.0.4.tar.gz 2357414 BLAKE2B 9c3368d745d30f57b313e9b5351714097403e67519a4582dd5658cb97152f4ad37c32cb9e6f47c13f199c75e9bd42c529ef0d352f363da815f486c90c305779c SHA512 9073df49639045319e971a76eb3769c184617e179694981ad88ddd02382b4ca50ff92c26239757022270e0a0a4115c8b70ca7efeab6ff2dee78e90a7de1bf5a7
 DIST seafile-client-7.0.5.tar.gz 2359779 BLAKE2B 58fa532668859c4413bb931146a58eb51ede7c1faa3ea2a9cafb94be44a8819261570ef16cc9f62f584f9ff483aa0832a63a99bb53380dd9efa4f82d25dea553 SHA512 b79cde3f98ebb1a0cb1035131cf95735f01c63bcb4bb658d5c40c36cdf85705d2c6fd2f7787fbdb3d965a25d60ee98e020e33c4f9423295c7ef1ab36ee5c7c27
+DIST seafile-client-7.0.7.tar.gz 2365877 BLAKE2B 60d4abf578de98b82f5107ba0c638c6e74bd62e71310d70b0369b380102cc749fe12cbb371ad12129f21e5ed020f21d1fde26b8fab0405033bd86eabb1f5990b SHA512 80d0a711e72eb39c65ae4a7cbfc576a10c4f6b30b7cf81ad7b2c0831e1a505fa500ce2ecd802dd79b80ef9e1ebb749585d1a82ac9999a0791f166300b1b0f6ed
```
Comment (juippis) [inline net-misc/seafile-client/Manifest:3]: This failed for me and I had to regenerate the manifest file.

**Comment by CaseOf** [comment]
@evdenis seems to want this PR to be redirected to this one https://github.com/gentoo/gentoo/pull/16961

**Pull request closed without merging**